### PR TITLE
API updates

### PR DESCRIPTION
**Pull request by divyaamunugama**
## Purpose
$subject

### CODE REVIEW

File: en/docs/apis/swaggerfiles/organization-role-mgt-api-v1.yaml [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
 info:
   title: WSO2 Identity Server - Organizational Role Management
-  contact:
-    name: WSO2
-    url: http://wso2.com/products/identity-server
-    email: iam-dev@wso2.org
-  license:
-    name: Apache 2.0
-    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  description: This is the RESTful API for managing organizational roles in CIAM Cloud. This API allows users to create, update, retrieve and delete organization roles.
```
Comment (nilminiwso2) [inline en/docs/apis/swaggerfiles/organization-role-mgt-api-v1.yaml:5]:
```suggestion
  description: This is the RESTful API for managing organizational roles in CIAM Cloud. This API allows users to create, update, retrieve, and delete organization roles.
```

File: en/docs/apis/swaggerfiles/organization-mgt-api-v1.yaml [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
   title: WSO2 Identity Server - Organization Management API Definition
-  description: This document specifies a **Organization Management RESTful API** for **WSO2 Identity Server**.
-  contact:
-    name: WSO2
-    url: http://wso2.com/products/identity-server/
-    email: iam-dev@wso2.org
-  license:
-    name: Apache 2.0
-    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  description: This is the RESTful API for organization management in CIAM Cloud. This API allows users to create, update, retrieve and delete organizations. This API also allows to enable and disable sharing of applications within an organization.
```
Comment (nilminiwso2) [inline en/docs/apis/swaggerfiles/organization-mgt-api-v1.yaml:5]:
```suggestion
  description: This is the RESTful API for organization management in CIAM Cloud. This API allows users to create, update, retrieve, and delete organizations. This API also allows to enable and disable sharing of applications within an organization.
```

File: en/docs/apis/swaggerfiles/organization-mgt-api-v1.yaml [hunk 3]
```diff
@@ -2,14 +2,14 @@
 openapi: 3.0.0
 info:
   title: WSO2 Identity Server - Organization Management API Definition
```
Comment (nilminiwso2) [inline en/docs/apis/swaggerfiles/organization-mgt-api-v1.yaml:4]: Will this be the tile of the doc?
It says `WSO2 Identity Server`

Comment (divyaamunugama) [reply en/docs/apis/swaggerfiles/organization-mgt-api-v1.yaml]: Updated to `WSO2 Private CIAM Cloud`

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**